x/market/keeper/deal_test.go

```diff
@@ -137,17 +137,22 @@ func (suite *dealTestSuite) TestSellOwnData() {
 	suite.Require().Equal(sellerBalance, reward)
 }
 
-//func (suite *dealTestSuite) TestVerify() {
-//	cert := makeTestCert()
-//
-//	validatorAddr, err := sdk.AccAddressFromBech32(cert.UnsignedCert.GetDataValidatorAddress())
-//	suite.Require().NoError(err)
-//	suite.Require().Equal(newAddr, validatorAddr)
-//
-//	verify, err := suite.MarketKeeper.Verify(suite.Ctx, validatorAddr, *cert.UnsignedCert, cert)
-//	suite.Require().Equal(true, verify)
-//	suite.Require().NoError(err)
-//}
+func (suite *dealTestSuite) TestVerify() {
+	cert := makeTestCert()
+
+	validatorAddr, err := sdk.AccAddressFromBech32(cert.UnsignedCert.GetDataValidatorAddress())
+	suite.Require().NoError(err)
+	suite.Require().Equal(newAddr, validatorAddr)
+
+	account := suite.AccountKeeper.NewAccountWithAddress(suite.Ctx, validatorAddr)
+	err = account.SetPubKey(privKey.PubKey())
```

Comment:
Some of the codes are validated with DataValidator's Pubkey.

Reply:
Also, secp256k1 should use cosmos-sdk, not tendermint.
Because `account.SetPubKey` received only cosmos-sdk's secp256k1

Reply:
Thank you for solving issue #259! Register ```cryptocodec``` interface in ```suite.go``` solves this issue as @hansol-medi said.